http/obmc_shell.hpp

```diff
@@ -202,7 +174,7 @@ static std::map<crow::websocket::Connection*, std::shared_ptr<Handler>>
 inline void requestRoutes(App& app)
 {
     BMCWEB_ROUTE(app, "/bmc-console")
-        .privileges({{"ConfigureManager"}})
+        .privileges({{"OemIBMPerformService"}})
```

Comment:
As stated in the PR message, this isn't doing anything

Reply:
yes

Reply:
we like to change this right right now in this commit